feature-libs/product-configurator/rulebased/core/facade/configurator-place-order.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { OrderPlacedEvent } from '@spartacus/checkout/root';
+import { EventService } from '@spartacus/core';
+import { ConfiguratorActions } from '../state/actions/index';
+import { StateWithConfigurator } from '../state/configurator-state';
+
+@Injectable({ providedIn: 'root' })
+export class ConfiguratorPlaceOrderService {
+  constructor(
+    protected store: Store<StateWithConfigurator>,
+    protected eventService: EventService
+  ) {}
+
+  init(): void {
+    this.eventService.get(OrderPlacedEvent).subscribe(() => {
+      this.store.dispatch(
+        new ConfiguratorActions.RemoveCartBoundConfigurations()
+      );
```

Comment:
Perhaps consider extracting this logic 

```
this.store.dispatch(
        new ConfiguratorActions.RemoveCartBoundConfigurations()
);
```

in a facade function if other code might want to call that directly ( and of course call that function within the  OrderPlacedEvent listener).

Just a suggestion, your call if this makes sense.

Reply:
Ok, I see. From my point of view the event listener pattern is better, because it instantly makes clear what this is about. Changed that. Also agree that the actual trigger of the new action should be in the facade layer